Documentation/ceph-csi-drivers.md

```diff
@@ -140,14 +137,8 @@ Find a Ceph operator pod (in the following example, the pod is
 kubectl exec -ti -n rook-ceph rook-ceph-operator-7464bd774c-scb5c -- bash -c "ceph -c /var/lib/rook/rook-ceph/rook-ceph.config auth get-or-create-key client.kubernetes mon \"allow profile rbd\" osd \"profile rbd pool=rbd\""
 ```
 
-Then create a Secret using admin and `kubernetes` keyrings:
-
-In [secret](../cluster/examples/kubernetes/ceph/csi/example/rbd/secret.yaml),
-you need your Ceph admin/user password encoded in base64.
-
-Run `ceph auth ls` in your rook ceph operator pod, to encode the key of your
-admin/user run `echo -n KEY|base64`
-and replace `BASE64-ENCODED-PASSWORD` by your encoded key.
+Run `ceph auth ls` in your rook ceph operator pod, to get the key of your
```

Comment:
```suggestion
Run `ceph auth get-key <user>` in your rook ceph operator pod, to get the key of your
```

Reply:
@leseb this is already done here https://github.com/rook/rook/pull/3358/files#diff-8be52e0c33719742b3b5ee58f5ae6f74R197
we are doing `ls` just to get the list of already created.

Reply:
The sentence implies (at least to me) that you know the name of the key so I don't understand why you would list all the keys to find it. Am I missing something?

Reply:
if you don't know the user name which already exists. we can list the users with his creds and permissions and we can choose which user details to use in secret.

Note:- this is not added in this PR, this PR is re-wording it